QuizApp/Screens/QuestionPage/QuizViewController.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  QuizViewController.swift
+//  QuizApp
+//
+//  Created by Ana on 11/26/24.
+//
+
+import UIKit
+
+final class QuizViewController: UIViewController {
+    
+    // MARK: - Properties
+    private let headerView = QuizHeaderView()
+    private let progressView = QuizProgressView()
+    private let questionView = QuizQuestionView()
+    
+    private lazy var answersTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.backgroundColor = .clear
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(
+            AnswerCell.self,
+            forCellReuseIdentifier: AnswerCell.identifier
+        )
+        tableView.separatorStyle = .none
+        tableView.isScrollEnabled = false
```

Comment:
why you are setting tableView.isScrollEnabled = false, what if you have more items